html/semantics/forms/the-input-element/hidden-charset-case-sensitive.html

```diff
@@ -19,7 +20,8 @@
     const params = new URL(event.data).searchParams;
 
     assert_true(params.get("_charset_") != "", "lowercase valid");
```

Comment:
Can you change this and the others to assert_equals with an actual expected value?

Reply:
done